genie-agent/src/main/java/com/netflix/genie/agent/execution/ExecutionAutoConfiguration.java

```diff
@@ -221,13 +222,15 @@ CreateJobDirectoryStage createJobDirectoryStage(final JobSetupService jobSetupSe
 
     /**
      * Create a {@link RelocateLogFileStage} bean if one is not already defined.
+     *
+     * @param agentLogManager the agent log manager
      */
     @Bean
     @Lazy
     @Order(70)
```

Comment:
This is minor but you may want to make these orders public static variables that people can reference if they want to ensure their bean is higher or lower precedence.